### ISSUE TITLE
False positive for Document.aspx in link

### ISSUE THREAD

**Issue by adam-boduch** [voku/anti-xss#95]
#### What is this feature about (expected vs actual behaviour)?

False positive for URL in `href` attribute removes whole URL if it contains `Document.aspx`.

#### How can I reproduce it?

```
$antiXss = new \voku\helper\AntiXSS();
$antiXss->xss_clean('<a href="http://google.com/Document.aspx">long link</a>');
```

**Comment by voku** [comment]
:-/ We currently disallow different JS keywords in `href` because it's hard to detect stored XSS attacks. But I can make the keywords configurable so that you can allow `document.`, what do you think?

**Comment by voku** [comment]
This didn't need much time, but I do not have that many free times with a little child and a baby here at home. (:child: && :baby:)

Here is an example (available in version 4.1.37): https://github.com/voku/anti-xss/commit/e1c6e822a4bd3cf00489a53fa184b0f7e4ab1899#diff-742333ddea22a32dd5f826ea678fffeb11dbee1945efa6b969cae65f8a0ab4e3R1502

**Comment by adam-boduch** [comment]
Thank you.
